src/configure/auto_outlook.rs

```diff
@@ -56,6 +57,7 @@ pub unsafe fn outlk_autodiscover(
             break;
         }
 
+        out_null = false;
```

Comment:
i think this assumes a result too soon.

out_null should only be disabled when there is really a valid configuration parsed. i think this is below after the condition checking there is no redirect:
```
if !(!outlk_ad.config[5].is_null()
            && 0 != *outlk_ad.config[5usize].offset(0isize) as libc::c_int) 
```

Reply:
Thanks. Actually it should be tested, but need to find some server with outlook autodiscovery for that.